packages/react-ssr/scripts/build.ts

```diff
@@ -30,7 +30,13 @@ const createReactWrapperModules = async ({ entryPoints, distRoot }) => {
     const enums = parseEnums(componentsTypeSource);
 
     let i = 0;
-    for (const importPath of entryPoints) {
+    for (let importPath of entryPoints) {
+
+      // Build on Window: File prefix required
+      if ( importPath.startsWith( "C:\\" ) ) {
```

Comment:
Nice catch! Would you be able to modify this to be able to catch any drive letter? Not every Windows user is guaranteed to have this installed on a C drive.

Reply:
It is now able to catch any drive letter.